styles.css

```diff
@@ -0,0 +1,1234 @@
+}
+
+.speaker-details {
+  flex: 80%;
+  margin-left: 8px;
+}
+
+.speaker-details .name {
+  font-size: 15px;
+  font-weight: 600;
+  margin: 2px auto;
+}
+
+.speaker-details .state {
+  font-size: 8px;
+  color: orangered;
+  margin: 2px auto;
+}
+
+.speaker-details .repute {
+  font-size: 8px;
+  font-weight: 500;
+}
+
+.break {
+  width: 10%;
+  background-color: lightgray;
+  height: 1px;
+  margin: 2px;
+}
+
+.more {
+  height: 5vh;
+  width: 100%;
+  display: flex;
+  justify-content: space-around;
+  text-align: center;
+  border: 1px solid lightgray;
+  border-radius: 5px;
+  cursor: pointer;
+  transform: translateY(39vh);
+}
+
+.more .bx {
+  color: orangered;
+  position: relative;
+  top: 60%;
+}
+
+/* about section */
+.about-headline {
+  height: 75vh;
+  width: 100%;
+  margin: 15px auto;
+}
+
+.about-headline .headline-container {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  min-width: 282px;
+  margin: 0.5rem auto;
+  align-content: center;
+  align-items: center;
+  height: 35vh;
+}
+
+.headline-container p {
+  font-size: 18px;
+  text-align: center;
+  color: orangered;
+  margin-bottom: 8px;
+}
+
+.headline-container h1 {
+  font-size: 32px;
+  color: orangered;
+  font-weight: 700;
+  text-align: center;
+}
+
+.about-headline .desc {
+  font-size: 13px;
+  background-color: #fff;
+  font-weight: 500;
+  line-height: 15px;
+  align-items: center;
+  display: flex;
+  width: 85%;
+  height: 25vh;
+  margin: auto auto;
+  text-align: center;
+}
+
+.about-headline .contact {
+  width: 100%;
+  color: grey;
+  text-align: center;
+  background-image: url('images/main-bg-img.png');
+  height: 22vh;
+}
+
+.hero p {
+  color: white;
+  font-size: 15px;
+  margin: 0;
+}
+
+.about-headline .contact p {
+  font-size: 14px;
+  width: 90%;
+  height: 100%;
+  margin: 0 auto;
+  padding-top: 5%;
+  font-weight: 400;
+}
+
+.summit-cover {
+  width: 100%;
+  margin: 1rem auto;
+}
+
+.soccer-summit-info {
+  width: 90%;
+  height: 60vh;
+  margin: 0 auto;
+  margin-top: -0.9rem;
+}
+
+.soccer-summit-info h2 {
+  font-size: 20px;
+  line-height: 28px;
+  color: #000;
+  text-align: center;
+  top: 1rem;
+  width: 90%;
+  height: 3vh;
+  margin: 0 auto;
+  margin-top: 1.5rem;
+  padding-top: 2.7rem;
+}
+
+.soccer-summit-info .underline {
+  width: 15%;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  top: 1.5rem;
+  margin: 0 auto;
+  margin-top: 0.7rem;
+  margin-bottom: 1.2rem;
+  height: 2px;
+  background-color: orangered;
+}
+
+.soccer-summit-info .desc {
+  font-size: 12px;
+  font-weight: 400;
+  margin: 1.5rem auto;
+  padding-top: 3%;
+  height: 15vh;
+}
+
+.soccer-summit-info svg {
+  width: 100%;
+  margin: 1rem auto;
+  height: 35vh;
+  position: relative;
+  top: -35%;
+}
+
+.breakline {
+  height: 1px;
+  position: relative;
+  top: -5.5rem;
+  width: 100vw;
+  margin-top: 0.5rem;
+  background-color: lightgrey;
+}
+
+.past-cover {
+  width: 100vw;
+  height: 100vh;
+}
```

Comment:
Great design @marvin-nyalik 🤝

- [ ] I would suggest that you center these images properly so that your design may replicate the design provided.
![mobile images](https://github.com/marvin-nyalik/SoccerFestival/assets/63917725/5623a275-5094-4c7e-87ff-6ef4c601a20b)